js/index.js

```diff
@@ -1,7 +1,88 @@
-// Iteration 1: Names and Input
+// Iteration 1
 
+let hacker1 = "Alberto";
+let hacker2 = "Alberto";
 
-// Iteration 2: Conditionals
+//console.log(`Driver\'s name is: ${hacker1}`);
+//console.log(`Navigator\'s name is: ${hacker2}`);
 
+// Iteration 2
 
-// Iteration 3: Loops
\ No newline at end of file
+if (hacker1.length > hacker2.length) {
+  console.log(
+    `The driver has the longest name, it has ${hacker1.lenght} characters`
+  );
+} else if (hacker2.length > hacker1.length) {
+  console.log(
+    `It seems that the navigator has the longest name, it has ${hacker2.length} characters`
+  );
+} else {
+  console.log(
+    `Wow, you both have equally long names, ${hacker1.length} characters!`
+  );
+}
+
+// Iteration 3
+
+//creamos un loop por cada carácter en el nombre de hacker1
+let upperHack = "";
+
+for (let i = 0; i < hacker1.length; i++) {
+  upperHack += hacker1[i].toUpperCase() + " ";
+}
+console.log(upperHack);
+
+// imprimimos el nombre del navegador al reverse usando un loop "negativo"
+
+let reverse = "";
+
+for (let i = hacker2.length - 1; i >= 0; i--) {
+  reverse += hacker2[i];
+}
+
+console.log(reverse);
+
+// Alphabetical order
+
+if (hacker1.localeCompare(hacker2) < 0) {
+  console.log("The driver's name goes first.");
+} else if (hacker1.localeCompare(hacker2) > 0) {
+  console.log("Yo, the navigator goes first definitely.");
+} else {
+  console.log("What?! You both have the same name?");
+}
```

Comment:
Good using the localeCompare() method.
You could also have used a loop here in order to check every character (like Uros showed this morning)